spec/factories/order_forms.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :order_form do
+    user_id {1}
+    item_id {1}
```

Comment:
```suggestion
    user_id {1}
    item_id {1}
```
外部キーを直接FactoryBotに記載するのはいい実装とは言えません。
今回購入において商品やユーザーを用意する必要があるのであれば、
order_spec.rbのbefore内でuserやitemのFactoryBotから生成しましょう。
理由としては以下になります。

```
①フォームオブジェクトはアソシエーションが使えないため、バリデーション にitem_idを記載している。

②実際のitem_idにはそれぞれのitem_idの情報が入ることになる。

③ですが、「 item_id { 1 }」のように記述してしまうと、
　常に「idの情報が1」の状態でテストが実行されることになり、
　実際の挙動と異なる。
```

ご不明な点がありましたら、コネクトにてメンターにご相談ください！